README.md

```diff
@@ -131,7 +131,7 @@ Follow the steps below for deploying the application:
 
 * Once viewing the application, click the `Runtime` option on the menu and navigate to the `Environment Variables` section.
 
-* Update the `CLASSIFIER_ID`, and `NATURAL_LANGUAGE_CLASSIFIER_IAM_APIKEY` variables with your `Model ID` from [Step 4](#4-train-the-nlc-model)  and NLC API key from [Step 2](#2-create-ibm-cloud-services). Click `Save`.
+* Update the `CLASSIFIER_ID`, and `NLC_IAM_APIKEY` variables with your `Model ID` from [Step 4](#4-train-the-nlc-model)  and NLC API key from [Step 2](#2-create-ibm-cloud-services). Click `Save`.
```

Comment:
you should use the default service name as the prefix so that the SDK can work as designed.  Don't abbreviate it.
https://github.com/watson-developer-cloud/python-sdk/blob/master/ibm_watson/natural_language_classifier_v1.py#L44

Reply:
Also I think the latest SDK will pick up <DEFAULT_NAME>_APIKEY, so you should drop the _IAM part of the name (I think in some versions they both worked).